### ISSUE TITLE
Add transparent Tensor (non-numpy) input to `evaluate`.

### ISSUE THREAD

**Issue by TimZaman** [fchollet/keras#7757]
Keras TOT (2.0.8):

Works:
`loss = model.evaluate(None, None, steps=steps)`

Suggestion conform current Tensor-input of `def fit(..)`:
`loss = model.evaluate(steps=steps)`

@athundt @ahundt Opinion?

**Comment by ahundt** [comment]
Yes that sounds good to me, it should be easy to add `None` as the default to those two evaluate parameters.
